docs/scripts/object_orientated_programming_poc/oop_poc_file.py

```diff
@@ -12,16 +12,24 @@
 7. Copying a file
 8. Storing an activity to a file
 9. Retrieve an activity from a file
+10. Bind a JSON schema to files and validate its contents
```

Comment:
I apologize that I did not catch this earlier, but these files will eventually be removed in favor of the tutorial pages instead. The intention of these files was to give the very first iteration on what tutorial scripts could look like.

Take a look at this confluence page that covers tutorial creation: https://sagebionetworks.jira.com/wiki/spaces/SYNPY/pages/3864690727/SYNPY-1549+Python+Client+New+Service+Implementation+SOP#Tutorials

Reply:
@linglp we will want to be careful adding lines of code to these, because we actually link specific lines of these tutorial code in the docs.